server/lib/device/light/light.command.js

```diff
@@ -15,6 +15,10 @@ const { DEVICE_FEATURE_CATEGORIES, DEVICE_FEATURE_TYPES } = require('../../../ut
 async function command(message, classification, context) {
   try {
     const devices = await this.getLightsInRoom(context.room);
+    if (devices.length === 0) {
```

Comment:
I'm not sure that doing the check here is enough ! 

If the device found contains a deviceFeature category = light but there is no type = binary, no lights will be turned on/off and we will not print a message.

What I would do instead is having a variable like "let lightAffected = 0;"

And inside the Promise.map, each time a light is turned on/off, incrementing the counter.

At the end of try {}, if lightAffected = 0, send the message "light.not-found".

What do you think?

Reply:
You're right, I'll made the update

Reply:
Thanks!

Reply:
Done 👌